src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -3,10 +3,15 @@
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
 
 public class DatabaseManager {
     private static final String PORT = "5432";
     private static final String SERVER = "localhost";
+    private static final String TABLE_NAMES_QUERY = "SELECT relname FROM pg_stat_user_tables ORDER BY relname";
+    private static final String TABLE_ROWS_COUNT_QUERY = "SELECT count(*) FROM ?";
+    private static final int VALID_TIMEOUT = 15;
     private Connection connection;
```

Comment:
static and non static blocks should be separate by blank line (http://bit.ly/2j2qPE1)

Reply:
Of course you are right.